test/WebJobs.Script.Tests/Helpers/SimpleWebTokenTests.cs

```diff
@@ -56,6 +57,39 @@ public void CreateTokenShouldCreateAValidToken()
             Assert.Equal($"exp={timeStamp.Ticks}", decrypted);
         }
 
+        [Fact]
+        public void ValidateTokenUsesContainerEncryptionKeyIfAvailable()
+        {
+            var containerEncryptionKey = TestHelpers.GenerateKeyBytes();
+            var containerEncryptionStringKey = TestHelpers.GenerateKeyHexString(containerEncryptionKey);
+
+            var websiteAuthEncryptionKey = TestHelpers.GenerateKeyBytes();
+            var websiteAuthEncryptionStringKey = TestHelpers.GenerateKeyHexString(websiteAuthEncryptionKey);
+
+            var timeStamp = DateTime.UtcNow.AddHours(1);
+
+            Environment.SetEnvironmentVariable(EnvironmentSettingNames.ContainerEncryptionKey, containerEncryptionStringKey);
+            Environment.SetEnvironmentVariable(EnvironmentSettingNames.WebSiteAuthEncryptionKey, websiteAuthEncryptionStringKey);
+
+            var token = SimpleWebTokenHelper.CreateToken(timeStamp, containerEncryptionKey);
+            Assert.True(SimpleWebTokenHelper.TryValidateToken(token, new SystemClock()));
+        }
+
+        [Fact]
+        public void ValidateTokenUsesWebsiteEncryptionKeyIfContainerEncryptionKeyNotAvailable()
```

Comment:
Can you follow the _ naming pattern we use for other tests to make this more readable? E.g. ValidateToken_UsesWebsiteEncryptionKey_IfContainerEncryptionKeyNotAvailable. Same for other tests.

Reply:
Other tests in this file were using camel case. Will switch to _